.doc_gen/metadata/kinesis_metadata.yaml

```diff
@@ -42,8 +50,32 @@ kinesis_DescribeStream:
               snippet_tags:
                 - python.example_code.kinesis.KinesisStream.class
                 - python.example_code.kinesis.DescribeStream
+    Rust:
+      versions:
+        - sdk_version: 1
+          github: .rust_alpha/kinesis
+          excerpts:
+            - description:
+              snippet_tags:
+                - kinesis.rust.describe-stream
   services:
-    kinesis: {DescribeStream}
+    kinesis: { DescribeStream }
+kinesis_ListStreams:
+  title: Lists information about one or more &AK; streams using an &AWS; SDK
+  title_abbrev: List streams
+  synopsis: list &AK; streams.

```

Comment:
I'd move this around to shorten the title and put extra info in the synopsis. Something like:
```
title: List &AK; streams using an &AWS; SDK
synopsis: list information about one or more &AK; streams.
```

Reply:
Done.